Formula/minikube.rb

```diff
@@ -0,0 +1,28 @@
+class Minikube < Formula
+  desc "Tool that makes it easy to run Kubernetes locally"
+  homepage "https://kubernetes.io/docs/getting-started-guides/minikube/"
+  url "https://github.com/kubernetes/minikube.git",
+    :tag => "v0.26.1",
+    :revision => "6ded2b647d1b1f62100c630bcfcc1363c631ce2d"
+
+  # Docker required for non-Linux builds
```

Comment:
Homebrew is for macOS. Remove `unless MacOS.version == Version::NULL` and comment

Reply:
This formula is meant to work on Linuxbrew as well, how is that normally handled?

Reply:
Linuxbrew will handle the changes for Linux